### DISCUSSION TITLE
How to reuse Nuxt web site in other project?

### DISCUSSION THREAD

**Discussion by mrFANRA** [nuxt/nuxt#20876] [Questions] [answered]
Hi! 

I have one nuxt website. Now I need make few identical sites for other clients based on first\base project. Different is only in design, all functionality is same. In the end point I want update functionality in base project, and all clients sites updating by dependencies with git.

My experience in nuxt is low. But in my opinion it is looks lite this:
1. Add base project as dependency to new.
2. Import base pages in any new pages.

But when pages is importing, they start import all theys dependencies. It is good, but build fail with exception:
ReferenceError: defineNuxtPlugin is not defined

It is happens when one of page from base project using plugin.

Help me please find good way for my issue.

**Comment by ndragun92** [comment, marked as answer]
I think this is what you are looking for.

https://www.youtube.com/watch?v=HvTui3P0w-o&ab_channel=LearnVue

**Comment by maou-shonen** [comment]
https://nuxt.com/docs/getting-started/layers#layers
